dev/com.ibm.ws.jsp.2.3_fat/test-applications/TestServlet.war/resources/exception.jsp

```diff
@@ -0,0 +1,12 @@
+<%--
+    Copyright (c) 2023 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+--%><% throw new Exception("Exception from JSP"); %>
```

Comment:
new line

Reply:
This file cannot end with a new line otherwise it causes an unaccessible code error.